.github/workflows/pull_request.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       - name: Build the Docker image
         run: |
           echo "ACTNOW_SECRET_KEY=${{ secrets.ACTNOW_SECRET_KEY }}" >> .env
+          echo "ACTNOW_SENTRY_DSN=${{ secrets.ACTNOW_SENTRY_DSN }}" >> .env
```

Comment:
`SECRET_KEY` is required why `SENTRY_DSN` is optional. Why this change?

Reply:
It was raising an exception because of not being set. I've removed this part and updated the code to make it optional.